repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -465,6 +498,13 @@ class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMo
       originalResponse.flushBuffer()
     }
 
+    if (sentError) {
```

Comment:
I think you are fine. I missed where commit was our method not part of the original contract, so i was mentally applying the meaning of commit of the response to the stream contract.